runtime/moonbase/src/xcm_config.rs

```diff
@@ -484,7 +496,11 @@ impl Into<Option<Location>> for AssetType {
 	fn into(self) -> Option<Location> {
 		match self {
 			Self::Xcm(location) => {
-				xcm_builder::WithLatestLocationConverter::convert_back(&location)
+				let versioned = xcm::VersionedLocation::V3(location);
+				match versioned.into_version(5) {
```

Comment:
```suggestion
				match versioned.into_version(xcm::latest::VERSION) {
```

Reply:
Addressed in https://github.com/moonbeam-foundation/moonbeam/pull/3224/commits/e22f72e3495f76eab43931bab013c1ce3140db35